src/gui/mainwindow.h

```diff
@@ -52,9 +52,9 @@ class TransferListWidget;
 class TransferListFiltersWidget;
 class PropertiesWidget;
 class StatusBar;
-class TorrentCreatorDlg;
+class TorrentCreatorDialog;
```

Comment:
you might want to sort this list, either now or in another PR.

Reply:
I'll do it in the next PR in a coding style commit along with a few other things i've found (small commit :)